playroom/snippets.tsx

```diff
@@ -1615,6 +1615,55 @@ const heroSnippets = [
     },
 ];
 
+const loaderSnippets = [
+    {
+        name: 'Loader with url',
```

Comment:
maybe include `Loader with url (internal use)` or something like that to avoid confusions?